src/main/java/lotto/View/Output.java

```diff
@@ -0,0 +1,86 @@
+package lotto.View;
+
+import static lotto.View.OutputMessage.Enter_Bonus_Numbers;
+import static lotto.View.OutputMessage.Enter_Lotto_Numbers;
+import static lotto.View.OutputMessage.Enter_Purchase_Mount;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import lotto.Model.Prize;
+import lotto.Model.ScoreBoard;
+import lotto.Model.LottoData;
+
+public class Output {
+    private static final String DIVIDE_POINT =", ";
+    private static final String START_PRINT_POINT ="[";
+    private static final String END_PRINT_POINT ="]";
+    public static void printLottos(ArrayList<LottoData> lottos){
+
+        System.out.printf(OutputMessage.PURCHASE_COUNT.getMessage(),lottos.size());
+
+        lottos.stream()
+                .map(LottoData::getNumbers)
+                .forEach(Output::printLotto);
+        System.out.println();
+    }
+
+    public static void printScores(ScoreBoard roundScore){
+            System.out.println(OutputMessage.TOTAL_PRIZE_TITLE.getMessage());
+
+            for (Prize prize : Prize.values()) {
+                int prizeCount = roundScore.getScoreByRank(prize);
+                printScore(prize,prizeCount);
+            }
+    }
+
+
+    public static void printError(String errorMessage){
+        System.out.println(errorMessage);
+    }
```

Comment:
```suggestion
    public static void printError(Exception error){
        System.out.println(error.getMessage());
    }
```
와 같이 바꿔보면 어떨까요?? 이렇게 하면 외부에서는 간단히 `OutputVIew.printError(error)` 로 간단히 호출해줄 수 있을 것 같아요!

Reply:
Error를 매개변수로 넘길 방법을 몰랐는데 이렇게 간단한거였군요 !! 감사해요 !!

Reply:
> Error를 매개변수로 넘길 방법을 몰랐는데 이렇게 간단한거였군요 !! 감사해요 !!

Error를 매개변수로 넘길 방법을 몰랐는데 이렇게 간단한거였군요 !! 감사해요 !!

Reply:
> 와 같이 바꿔보면 어떨까요?? 이렇게 하면 외부에서는 간단히 `OutputVIew.printError(error)` 로 간단히 호출해줄 수 있을 것 같아요!

Error를 매개변수로 넘길 방법을 몰랐는데 이렇게 간단한거였군요 !! 감사해요 !!